inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -105,6 +106,10 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     manager.register_pass<ngraph::pass::GroupConvolutionBackpropDataMultiplyFusion>();
     manager.register_pass<ngraph::pass::ConstantFolding>();
 
+    manager.register_pass<ngraph::pass::ConvertNMS1ToNMS5>();
```

Comment:
Can we have an "umbrella" transformation which will run all upgrading transformation NMS1->NMS5, NMS3->NMS5 etc so here we have just one transformation call?

Reply:
Done.